indico/MaKaC/webinterface/pages/abstracts.py

```diff
@@ -914,7 +914,7 @@ def _getTrackItemsHTML( self ):
 
     def _getSessionItemsHTML( self ):
         items = [ i18nformat("""<option value="conf">--_("no session")--</option>""")]
-        for session in self._conf.getSessionList():
+        for session in sorted(self._conf.getSessionList(), key=lambda s: s.getTitle()):
```

Comment:
OK, I will keep `methodcaller` in mind, thx.
I will update the code with your latest suggestion.